### PR TITLE
python38-devel: update version to 3.8.0-b3, hashes and filesize.

### PR DESCRIPTION
**Pull request by Themanwithoutaplan**
###### Type(s)
- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.14.6 18G87
Xcode 10.3 10G8

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?

Hope I've done things properly this time.

### CODE REVIEW

File: lang/python38-devel/Portfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ PortGroup           select 1.0
 
 name                python38-devel
 
-version             3.8.0b2
+version             3.8.0b3
 epoch               1
 revision            1
```
Comment (mf2k) [inline lang/python38-devel/Portfile]: The `revision` needs to be set back to 0 when increasing the `version`.

**Comment by reneeotten** [comment]
Next request will be to squash the commits, so it would be good if you can take care of that now

**Comment by Themanwithoutaplan** [comment]
> Next request will be to squash the commits, so it would be good if you can take care of that now

I'd like to if I knew how. Is this possible in the GitHub UI?

File: lang/python38-devel/Portfile (hunk 1 above)
Comment (pmetzger) [reply lang/python38-devel/Portfile]: Yup.

**Comment by pmetzger** [comment]
> I'd like to if I knew how. Is this possible in the GitHub UI?

Not really. But I think I'm willing to use the squash-and-merge functionality as this is simple enough.

**Pull request merged**